src/Flexdis86/Disassembler.hs

```diff
@@ -500,12 +500,13 @@ simplePrefixes mnem allowed
 
 simplePrefixBytesFuns :: [(String, (Word8, PrefixAssignFun))]
 simplePrefixBytesFuns =
-  [ ("lock",  (0xf0, set prLockPrefix LockPrefix))
-  , ("repnz", (0xf2, set prLockPrefix RepNZPrefix))
-  , ("repz",  (0xf3, set prLockPrefix RepZPrefix))
-  , ("rep",   (0xf3, set prLockPrefix RepPrefix))
-  , ("oso",   (0x66, set prOSO True))
-  , ("aso",   (0x67, set prASO True))
+  [ ("lock",    (0xf0, set prLockPrefix LockPrefix))
+  , ("repnz",   (0xf2, set prLockPrefix RepNZPrefix))
+  , ("repz",    (0xf3, set prLockPrefix RepZPrefix))
+  , ("rep",     (0xf3, set prLockPrefix RepPrefix))
+  , ("oso",     (0x66, set prOSO True))
+  , ("aso",     (0x67, set prASO True))
+  , ("notrack", (0x3e, set prNoTrack True))
```

Comment:
I'm not sure if it makes sense to lump `notrack` with these other prefixes, which are part of the "legacy prefixes" set (see [here](https://wiki.osdev.org/X86-64_Instruction_Encoding#Legacy_Prefixes)). For clarity's sake, I'd prefer to put `notrack` into its own, separate definition, as it's an entirely different thing.

Reply:
Ah, indeed it is not part of the legacy. I will see if I can put it elsewhere and keep things working.